config/application.rb

```diff
@@ -214,8 +215,9 @@ class Application < Rails::Application
     # Add apidoc hash in headers for smarter caching
     config.middleware.use Apipie::Middleware::ChecksumInHeaders
 
-    # Add telemetry
+    # Add telemetry and connection cleaner
     config.middleware.use Foreman::Middleware::Telemetry
+    config.middleware.use Foreman::Middleware::ConnectionCleaner
```

Comment:
ConnectionCleaner is very generic name, I'll not know in the future what it's doing just by looking at this.
Can we have `LibvirtConnectionCleaner` or at least a comment what it really does?